### PR TITLE
add whitelist configuration file

### PR DESCRIPTION
**Pull request by droslean**
/cc @petr-muller @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
@sttts this PR is adding a white-list of repositories that need to exist in the private organization to facilitate CVE fixes. We should identify repositories here that are not already synced over, so there are a couple of cases:

 - repositories that have ci-operator configuration but do not promote anything to ocp/4.foo
 - repositories that do not have ci-operator configuration

Do you mind providing a list of the repositories you know of that should be in this list? I assume the downstream forks of k8s components we vendor will all be here.

**Comment by sttts** [comment]
openshift/kubernetes-* is a good start.

File: core-services/openshift-priv/_whitelist.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+whitelist:
+  openshift:
+    - kubernetes
+    - kubernetes-namespace-reservation
```
Comment (sttts) [inline core-services/openshift-priv/_whitelist.yaml:4]: this one is not an upstream fork, but our own repo

Comment (droslean) [reply core-services/openshift-priv/_whitelist.yaml]: Thanks. removed.

**Pull request merged**